lib/src/HttpMessageBody.h

```diff
@@ -121,40 +121,38 @@ class HttpMessageStringViewBody : public HttpMessageBody
     }
     const std::string &getString() const override
     {
-        if (!bodyString_)
+        if (bodyString_.empty())
         {
             if (!body_.empty())
             {
-                bodyString_ =
-                    std::make_unique<std::string>(body_.data(), body_.length());
+                bodyString_ = body_;
             }
             else
             {
-                bodyString_ = std::make_unique<std::string>();
+                bodyString_.clear();
             }
         }
-        return *bodyString_;
+        return bodyString_;
     }
     std::string &getString() override
     {
-        if (!bodyString_)
+        if (bodyString_.empty())
         {
             if (!body_.empty())
             {
-                bodyString_ =
-                    std::make_unique<std::string>(body_.data(), body_.length());
+                bodyString_ = body_;
             }
             else
             {
-                bodyString_ = std::make_unique<std::string>();
+                bodyString_.clear();
```

Comment:
bodyString is empty, why need clear?

Reply:
You are right. This can be further simplified. Also can bodyString initialization be moved into constructor or the point is in lazy initialization to copy on demand? It's the same copy-pasted logic in const and nonconst methods.